src/scss/_variables.scss

```diff
@@ -68,3 +75,91 @@ $spacers: (
 $embed-responsive-aspect-ratios: ((16 9), (2 1));
 $enable-responsive-font-sizes: true;
 $rfs-factor: 4;
+
+//Global typography
```

Comment:
Its seems like a lot of variables just for one small component? $btn-padding-y-lg: $input-btn-padding-y-lg; //17.5px does not seems like a good approach to me. what happens if someone changes the value of these variables ?